rhel/matcher.go

```diff
@@ -28,12 +29,32 @@ func (*Matcher) Filter(record *claircore.IndexRecord) bool {
 func (*Matcher) Query() []driver.MatchConstraint {
 	return []driver.MatchConstraint{
 		driver.PackageModule,
-		driver.RepositoryName,
```

Comment:
Do you think we will suddenly match against a lot more rows than previously?

Reply:
The subset we get back from the DB will be bigger, but the repository matching logic is just moving the the `Vulnerable()` method in order to be able to do more complex comparisons than string matching.